languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,21 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rule id="ELLES_VERBE_TOUTES" name="elles verbe toutes" default="temp_off">
+              <pattern>
+                  <token postag="R.* f p" postag_regexp="yes"></token>
+                  <token postag="V.*" postag_regexp="yes"></token>
```

Comment:
At the end of a token, match etc., if you have`></token>`, it's always better to just write `/>`. In this case:

`<token postag="R.* f p" postag_regexp="yes"></token>` → `<token postag="R.* f p" postag_regexp="yes"/>`

etc.